lib/tests/integration_test/server/api_v1_ApiTest.h

```diff
@@ -37,6 +37,11 @@ class ApiTest : public drogon::HttpController<ApiTest>
     METHOD_ADD(ApiTest::formTest, "/form", Post);
     METHOD_ADD(ApiTest::attributesTest, "/attrs", Get);
     ADD_METHOD_VIA_REGEX(ApiTest::regexTest, "/reg/([0-9]*)/(.*)", Get);
+    METHOD_ADD(ApiTest::cacheTest, "/cacheTest", Get);
+    METHOD_ADD(ApiTest::cacheTest2, "/cacheTest2", Get);
+    ADD_METHOD_VIA_REGEX(ApiTest::cacheTestRegex,
+                         "/cacheTestRegex/[a-y]+",
```

Comment:
@an-tao Is there a way for the controller to automatically append it's path to a regex handler? It feels inconsistent and too me half an hour to figure out why I'm getting a 404.